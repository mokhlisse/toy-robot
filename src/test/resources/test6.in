PLACE 0,1,EAST
MOVE
MOVE
MOVE
MOVE
MOVE
RIGHT
MOVE
RIGHT
MOVE
REPORT