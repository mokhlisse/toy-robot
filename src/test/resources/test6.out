Output: 4,0,WEST